test/e2e/multi-stage.sh

```diff
@@ -27,6 +27,7 @@ os::cmd::expect_success "ci-operator ${namespace} --secret-dir ${PULL_SECRET_DIR
 os::cmd::expect_success "ci-operator ${namespace} --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target without-references --unresolved-config ${suite_dir}/config.yaml"
 os::cmd::expect_success "ci-operator ${namespace} --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target with-references --unresolved-config ${suite_dir}/config.yaml"
 os::cmd::expect_success "ci-operator ${namespace} --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target skip-on-success --unresolved-config ${suite_dir}/config.yaml"
+os::cmd::expect_code_and_text "ci-operator ${namespace} --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target timeout --unresolved-config ${suite_dir}/config.yaml" 1 fixme-error-message
```

Comment:
```suggestion
os::cmd::expect_code_and_text "ci-operator ${namespace} --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target timeout --unresolved-config ${suite_dir}/config.yaml" 1 DeadlineExceeded
```

Reply:
I was hoping to inlude a longer regexp showing more of the error message

Reply:
ci-operator won't keep too many info from the namespace it has created, see [logs in my test PR](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_ci-tools/1257/pull-ci-openshift-ci-tools-master-e2e/1313183218622533632/artifacts/e2e/e2e/container-logs/test.log):
```
  * could not run steps: step timeout failed: "timeout" test steps failed: ["timeout" pod "timeout-timeout" failed: the pod ci-op-vr5h54kv/timeout-timeout failed after 1h0m43s (failed containers: ): DeadlineExceeded Pod was active on the node longer than the specified deadline
```

Reply:
Those logs still have you waiting for an hour and seeing

```
2020/10/05 19:52:43 The namespace in which this test is executing has been deleted, cancelling the test...
```

Reply:
Ok, ed358a5be adds the `activeDeadlineSeconds` workaround for [rhbz#1885717][1], and updates this test-case to expect `cleanup: activeDeadlineSeconds 10s exceeded, deleting pod timeout-timeout`.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1885717